src/containers/MyNdla/components/UserProfileTag.tsx

```diff
@@ -0,0 +1,103 @@
+/**
+ * Copyright (c) 2023-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import styled from '@emotion/styled';
+import { fonts, colors, spacing } from '@ndla/core';
+import SafeLink from '@ndla/safelink';
+import { useTranslation } from 'react-i18next';
+import Avatar from './Avatar';
+import { useArenaUser } from '../arenaQueries';
+
+type UserProfileTagProps = {
+  displayName: string | undefined;
+  username: string | undefined;
+  affiliation: string | undefined;
```

Comment:
Kunne disse vore definert slik?
```suggestion
  displayName?: string;
  username?: string;
  affiliation?: string;
```

Reply:
spørs på hva en vil. sånn som det er nå så vil disse props kreves men komponenten vil ikke breake hvis det plutselig kommer en undefined verdi. Med ? så vil ikke props kreves og må da fylles inn manuelt, som jeg er ikke så glad i. men jeg kan endre det til det som dere foretrekker å bruke i prosjektet

Reply:
Ikkje så farlig. Eg berre tenkte det var det samme, men det er det jo tydeligvis ikkje. Der lærte eg nokke nytt i dag.